service/history/historyEngine.go

```diff
@@ -2303,13 +2377,36 @@ Update_History_Loop:
 
 		err = context.updateWorkflowExecutionAsActive(e.shard.GetTimeSource().Now())
 		if err == ErrConflict {
```

Comment:
this err `ErrConflict` is deprecated, please remove all for loop retry for this err

Reply:
Updated, Please double check if I understand you correctly.

Reply:
I tried to change
```
		err = context.updateWorkflowExecutionAsActive(e.shard.GetTimeSource().Now())
		if err == ErrConflict {
			continue UpdateHistoryLoop
		}
		return err
```
 to 
```
		return context.updateWorkflowExecutionAsActive(e.shard.GetTimeSource().Now())
```
but encountered several unit test failed. how about we change it separately?

Reply:
for this `ErrConflict` sure.